core/client/fs/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -462,4 +480,45 @@ public void unmount(AlluxioURI path, UnmountOptions options)
       mFileSystemContext.releaseMasterClient(masterClient);
     }
   }
+
+  /**
+   * Alluxio URIs in the Filesystem don't need a scheme or host. Warn the user and throw an
+   * exception if necessary.
+   */
+  protected static void UriCheck(AlluxioURI uri) {
+    boolean logWarning = false;
+    if (uri.hasScheme()) {
+      if (uri.getScheme().compareTo("alluxio") == 0
+          || uri.getScheme().compareTo("alluxio-ft") == 0) {
+        logWarning = true;
+      } else {
+        // Throw Exception
+        throw new IllegalArgumentException(
+            "Valid scheme values in filesystem URI operations are \"alluxio\" and \"alluxio-ft\"."
```

Comment:
alluxio-ft is deprecated so it's better not to suggest it any more. We can shorten this to something like "Either use the \"alluxio://\" scheme or no scheme at all". We should also report the offending scheme to the user

Reply:
Should I still perform the check for `alluxio-ft` though before throwing the error?

Reply:
I think so - in general we treat `alluxio-ft` scheme the same as `alluxio`.

Btw, there are constants defined in `alluxio.Constants` for the scheme names - `Constants.SCHEME` and `Constants.SCHEME_FT`